OIV-NL-QGIS/OIV_project
# Oiv mobiel (#194)

* add_werkvoorraad_layers

werkvoorraad layers added to project and added styling to distinguish edited features from a mobile application from normal ones

* Test pg_service file

* init werkvoorraad ui

included modifications v3.3.7

* setup execute modifications

first setup of executing the werkvoorraad modifications

* Werkvoorraad in oiv plugin

- check if there is Werkvoorraad for object
- further development of handling werkvoorraad acceptions or denials.

* Delete pg_service.conf
